qiskit/test/mock/backends/__init__.py

```diff
@@ -31,3 +31,4 @@
 from .johannesburg import FakeJohannesburg
 from .boeblingen import FakeBoeblingen
 from .rochester import FakeRochester
+from .fake_100q import FakeOpenPulse100Q
```

Comment:
Perhaps this should live in `test.mock` since `backends` currently seems reserved for a physical backend?